src/main/java/dev/jbang/cli/Export.java

```diff
@@ -22,6 +31,105 @@
 	}, description = "Force export, i.e. overwrite exported file if already exists", defaultValue = "false")
 	boolean force;
 
+	@CommandLine.Option(names = {
+			"--style" }, description = "Which style should the export be. Valid values: ${COMPLETION-CANDIDATES}", defaultValue = "local")
+	Style style;
+
+	enum Style {
+		local {
+
+			public int apply(Export export, Script script, Path outputPath) throws IOException {
+				// Copy the JAR or native binary
+				Path source = script.getJar().toPath();
+				if (export.nativeImage) {
+					source = getImageName(source.toFile()).toPath();
+				}
+
+				if (outputPath.toFile().exists()) {
+					if (export.force) {
+						outputPath.toFile().delete();
+					} else {
+						export.warn("Cannot export as " + outputPath + " already exists. Use --force to overwrite.");
+						return EXIT_INVALID_INPUT;
+					}
+				}
+
+				Files.copy(source, outputPath);
+				export.info("Exported to " + outputPath);
+				return EXIT_OK;
+			}
+		},
+		slim {
+			@Override
+			public int apply(Export export, Script script, Path outputPath) throws IOException {
+				// Copy the JAR or native binary
```

Comment:
I think this part until line 80 is the same as the `local` style, right? We should probably factor that out.